test/Microsoft.AspNetCore.Sockets.Tests/MapEndPointTests.cs

```diff
@@ -62,7 +65,7 @@ public void MapEndPointFindsAuthAttributeOnEndPoint()
         public void MapEndPointFindsAuthAttributeOnInheritedEndPoint()
         {
             var authCount = 0;
-            var builder = new WebHostBuilder()
+            using (var builder = new WebHostBuilder()
```

Comment:
does it make sense to move building the host to a separate function?

Reply:
They all have slightly different services and other settings

Reply:
Sounds reasonable.